sdk/textanalytics/ai-text-analytics/samples/javascript/recognizeLinkedEntities.js

```diff
@@ -17,7 +17,7 @@ const apiKey = process.env["TEXT_ANALYTICS_API_KEY"] || "<api key>";
 
 const documents = [
   "Microsoft moved its headquarters to Bellevue, Washington in January 1979.",
-  "Steve Ballmer stepped down as CEO of Microsoft and was succeeded by Satya Nadella.",
+  "Steve Ballmer stepped down as CEO of Microsoft and was succeeded by Satya Nadella."
```

Comment:
Are these the only sample changes with all of the breaking changes you have? Not sure, but seems a little curious to me

Reply:
Yes because I had already removed or never incorporated code that used the grapheme properties, so the `confidenceScores` changes are really the only things that are breaking here, and I don't show the score in all of the samples.

This did remind me to check the README samples, though, and I found some places that I missed.